e2e/action/drop_node.go

```diff
@@ -0,0 +1,36 @@
+package action
+
+import (
+	"log"
+	"math/rand"
+
+	"github.com/0xPolygon/pbft-consensus"
+	"github.com/0xPolygon/pbft-consensus/e2e"
```

Comment:
Do you think it's worth to move action to a separate package?
It seems we may face a cycle dependencies there.

Reply:
Personally, I find it cleaner this way. Actions are used exclusively for fuzz framework and therefore intention was to keep it in a separate package.

Reply:
The app would not compile if there are cycle deps here. I think it looks better than having everything within one package/file. Made everything more structured. Let's say if you want to add a new action, you just need to create one more file in the actions package and it would be clear and understandable. 

Regarding deps in this case: main -> command -> actions -> e2e. Everything is ok with this.